policies/registry.rego

```diff
@@ -5,38 +5,59 @@ package before_resolution
 # used by semantic conventions.
 
 # Helper to create attribute registry violations.
-attr_registry_violation(violation_id, group_id, attr_id) = violation {
-	violation := {
```

Comment:
Maybe we should consider adding an editorconfig here to avoid the tabs/space mixes.

Reply:
great point, following up in https://github.com/open-telemetry/semantic-conventions/pull/1286